index.html

```diff
@@ -31,9 +31,53 @@ <h1 class="title">Error</h1>
 };
     </script>
 
-    <script src="node_modules/three/build/three.js"></script>
-    <script src="node_modules/three/examples/js/controls/VRControls.js"></script>
-    <script src="node_modules/three/examples/js/effects/VREffect.js"></script>
-    <script src="build/embed.js"></script>
+    <script>
+      (function () {
+        function loadScript(src, options) {
+          var script = document.createElement('script');
+          script.src = src;
+          script.async = false;
+          document.head.appendChild(script);
+
+          if (typeof options !== 'undefined') {
```

Comment:
It would be better to assign the event handlers before adding the script to the document in case it loads very fast, e.g., because it is already in the cache.  Otherwise, the event handlers may not be called.